config.json

```diff
@@ -106,6 +106,22 @@
                 "MAX_LIMIT": 1,
                 "WINDOW_SIZE": 1440
             }
+        },
+        {
+            "ID": "PirateVerse",
+            "NAME": "PirateVerseTestnet",
+            "TOKEN": "PVF",
+            "RPC": "https://rpc-testnet.pirateverse.finance/",
+            "CHAINID": 56766,
+            "EXPLORER": "https://testnet.piratescan.io",
+            "IMAGE": "/pirateverse.png",
```

Comment:
It will be nice to have a URL for the image here, instead of accessing it from the public directory.